tests/QueryTest.php

```diff
@@ -451,4 +451,47 @@ public function testSuggest()
 
         $this->assertCount(5, $result['customer_name'][0]['options']);
     }
+
+    public function testRuntimeMappings()
+    {
+        // Check Elasticsearch version
+        $elasticsearchInfo = $this->getConnection()->get('/');
+        preg_match('/^(\d+)\.(\d+)\.(\d+)$/', $elasticsearchInfo['version']['number'], $matches);
```

Comment:
Can `version_compare()` be used here? It would be easier.

Reply:
Thanks, I was unaware of this function.